src/lib/codeSamples/convertRequestToHarRequest.ts

```
@@ -0,0 +1,112 @@
+  }
+
+  // Handle query parameters
+  try {
+    harRequest.queryString = query.map(([name, value]) => ({
+      name,
+      value,
+    }))
+
+    // Prevent duplication of query params
+    url.search = ''
+  } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(unused-imports/no-unused-vars)